tstest/integration/vms/vms_test.go

```diff
@@ -530,3 +530,17 @@ runcmd:
  - [ systemctl, enable, --now, tailscaled.service ]
  - [ curl, "{{.HostURL}}/myip/{{.Port}}", "-H", "User-Agent: {{.Hostname}}" ]
 `
+
+type testLogWriter struct {
```

Comment:
use tstest.FixLogs(t) instead: https://github.com/tailscale/tailscale/blob/v1.8.6/tstest/log.go#L18

Reply:
Oh, I guess you'll need to export (and document) some stuff instead.

Reply:
I can just yoink log.Output!